libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -342,7 +358,7 @@ AP_GPS_UBLOX::read(void)
                 _delay_time = 750;
             }
         } else {
-            _delay_time = 2000;
+            _delay_time = 4000;
```

Comment:
there is a new message yes. currently used is https://pixhawk.ethz.ch/mavlink/#GPS_INJECT_DATA  110 bytes
however recently created is https://pixhawk.ethz.ch/mavlink/#GPS_RTCM_DATA  180 bytes
the newer message along with mavlink 2 is the best solution however. as it has 0 trimming and would mean you can send 50 byte packets without the extra 130 byte waste etc.